netbox/ipam/tests/test_filtersets.py

```diff
@@ -851,6 +852,32 @@ def test_filter_address(self):
         params = {'address': ['2001:db8::1/64', '2001:db8::1/65']}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
+        # Check for valid edge cases. Note that Postgres inet type
+        # only accepts netmasks in the int form, so the filterset
+        # casts netmasks in the xxx.xxx.xxx.xxx format.
+        params = {'address': ['24']}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 0)
+        params = {'address': ['10.0.0.1/255.255.255.0']}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
+        params = {'address': ['10.0.0.1/255.255.255.0', '10.0.0.1/25']}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+
+        # Check for invalid input.
+        params = {'address': ['/24']}
+        with self.assertRaises(serializers.ValidationError) as cm:
+            self.filterset(params, self.queryset).qs.count()
+        self.assertRegex(cm.exception.detail['address'], r'^Invalid address.*')
+
+        params = {'address': ['10.0.0.1/255.255.555.0']}
```

Comment:
A comment would be helpful here. This took me _way_ too long to tell apart from line 860. :laughing: 
```suggestion
        params = {'address': ['10.0.0.1/255.255.555.0']}  # Invalid subnet mask
```

Reply:
Welp - updating and will change the invalid subnet mask to something easier to catch